src/Sulu/Bundle/CategoryBundle/Exception/CategoryNotFoundException.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CategoryBundle\Exception;
+
+class CategoryNotFoundException extends \Exception
+{
+    /**
+     * @var mixed
+     */
+    private $categoryIdentifier;
```

Comment:
I think we go with `Id` in all the other parts of Sulu, can you change that with all references in this class?